terms/qc/validCellBarcodeReads.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema",
-    "$id": "sage.annotations-qc.validBarcodeReads-0.0.1",
+    "$id": "sage.annotations-qc.validCellBarcodeReads-0.0.2",
```

Comment:
If you're changing the name of the key, you don't have to update the version number because the term will have never been registered under the new name.